rpm/centos8/scripts/build-agent

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
```

Comment:
Some scripts invoke sh and some bash. Can we choose one or the other? Preferably sh.

Reply:
The `<<<` in version requires `bash` over `sh`, but we can standardize on using `/bin/bash` for all of the scripts

Reply:
We've been going the other direction for portability purposes since not all systems have bash, especially the smaller ones.

Reply:
I understand the concern, but because we run these scripts within Dapper (and are using our Dapperfile), I am not sure portability is as much of a concern.

Reply:
Agreed. Just consistency in developer expectation. This is fine for now.